recipe/meta.yaml

```diff
@@ -37,7 +37,6 @@ requirements:
 
   run:
     - python
-    - {{ pin_compatible("numpy") }}
```

Comment:
Is it correct that the numpy dep got removed here?

Reply:
Yes, it is mentioned in the migrator message above:

> Additionally, we no longer need to use {{ pin_compatible("numpy") }} as a
run requirement - this has been handled for more than two years now by a
run-export on the numpy package itself. The migrator will therefore remove
any occurrences of this.

Reply:
:shrug: OK

Thanks!